Src/Support/GoogleApis.Auth.PlatformServices_Shared/OAuth2/PromptCodeReceiver.cs

```diff
@@ -41,7 +40,13 @@ public string RedirectUri
             var authorizationUrl = url.Build().ToString();
 
             Logger.Debug("Open a browser with \"{0}\" URL", authorizationUrl);
-            Process.Start(authorizationUrl);
+#if NETSTANDARD
+            Console.WriteLine("Please visit the following URL in a web browser, then enter the code:");
```

Comment:
The browser page displayed with the code, says exactly this:

Google
Please copy this code, switch to your application and paste it there:
<code>

So I'm not too sure what better wording could be used. If you have an improved wording, please let me know!